doc/article/technical_article.tex

```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
 
 \subsection{Vue.js}
 Vue.js is a progressive framework for building user interfaces.\cite{vuejs}
-Vue.js is used for trckr mainly for its simplicity and the rather shallow
-learning curve it provides to unexperienced developers. The features that Vue.js
-provides allow the creation of data structures that can easily be displayed in
-on a website. This and the ability to easily make calls to the backend make it a
-good fit for trckr.
+Vue.js is used mainly for its simplicity and the rather shallow learning curve
+it provides unexperienced developers. The features that Vue.js provides allow
+the creation of data structures that can easily be displayed on a website. This
+and the ability to easily make calls to the backend make it a good fit for
+trckr.
 
 \section{Results}
-The architecture was kept fairly simple, while still providing a clear seperation of concerns as illustrated in figure \ref{fig:architecture}, using Django with PostgreSQL in the backend and a
-frontend based around Vue.js.
+The architecture was kept fairly simple, while still providing a clear
+seperation of concerns as illustrated in figure \ref{fig:architecture}, using
+Django with PostgreSQL in the backend and a frontend based around Vue.js.
```

Comment:
You could already write "front-end" and "back-end" even though the figure has yet to be updated :-)